### ISSUE TITLE
Hitting escape releases the mouse... and closes whatever window you have open

### ISSUE THREAD

**Issue by benwiley4000** [felixrieseberg/windows95#26]
I forgot this is what the escape key used to do... so annoying! Maybe:

1. Use a different keyboard shortcut to release the mouse from the app
or
2. Preempt Windows using the event and stop capture propagation. Maybe if the user *really* needs to hit escape inside of Windows they can use a special keyboard shortcut.

**Comment by felixrieseberg** [comment]
Yeah, that's a legit issue. I'd love to see a PR that allows us to do mouse capturing without also sending input to the VM.

**Comment by benwiley4000** [comment]
Yeah the VM mouse is weird. Not the same issue but I did a spike on a possible PR to re-capture the mouse at the same coordinates as your host mouse when you click back in - I couldn't get it to work quite right, so I gave up.

**Comment by bradtchapman** [comment]
This issue should probably be closed since it's 4 years old, but I'll open a new issue / FR for the mouse release function.

**Comment by felixrieseberg** [comment]
Fixed in spirit and in fact: since v5.0 the mouse is seamless (no pointer lock in normal use), so Escape isn't doing double duty anymore. Since v5.0 the mouse is seamless: a VBMOUSE driver in the guest tracks your real cursor 1:1, so there's no pointer-lock capture to escape from in normal use (Escape still toggles capture for the rare app that needs relative mouse). The Machine menu can also send Esc, Ctrl+Alt+Del, Alt+F4, and Alt+Enter directly into the guest. Closing this out — give the current release a try.
